vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -255,7 +323,7 @@ public void setDatePlaceholder(String placeholder) {
     /**
      * Gets the placeholder string of the date field.
      *
-     * @return the {@code placeholder} property of the date picker
+     * @return the {@code placeholder} property of the date time picker
      */
     public String getDatePlaceholder() {
         return datePicker.getPlaceholder();
```

Comment:
This was correct before and is now incorrect since this gets the "date placeholder".

Reply:
Updated.